__tests__/articlesRoutes.tests.js

```diff
@@ -47,3 +47,54 @@ describe("GET /api/articles/:article_id", () => {
     });
   });
 });
+
+describe("GET /api/articles", () => {
+  describe("Happy path", () => {
+    it("should return a 200", async () => {
+      await request(app).get("/api/articles").expect(200);
+    });
+    it("should return an array of objects", async () => {
+      const {
+        body: { articles },
+      } = await request(app).get("/api/articles").expect(200);
+      expect(articles).toBeInstanceOf(Array);
+      expect(articles).not.toHaveLength(0);
+    });
+    it("should return a list of objects that match the article schema", async () => {
+      const {
+        body: { articles },
+      } = await request(app).get("/api/articles").expect(200);
+      articles.forEach((article) => {
+        expect(article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String || undefined),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String || undefined),
+          comment_count: expect.any(String),
```

Comment:
great job here, I'd also like to check that the articles are sorted by descending date (jest sorted is great)